src/Symfony/Component/HttpKernel/DataCollector/DataCollectorInterface.php

```diff
@@ -26,7 +26,7 @@ interface DataCollectorInterface
     /**
      * Collects data for the given Request and Response.
      */
-    public function collect(Request $request, Response $response, \Exception $exception = null);
+    public function collect(Request $request, Response $response/*, \Exception $exception = null*/);
```

Comment:
this must be updated to `Throwable` and a corresponding annotation should be added